### PR TITLE
Add email address to request certs script

### PR DESCRIPTION
**Pull request by mec**
The script we use to generate the VSP certificate signing request should
include the group email address.

It would also be helpful if they were put in a directory that is git ignored so 
developers like myself do not accidentally commit them!

### CODE REVIEW

**Comment by mec** [comment]
We are going to wait until we have complete a certificate request with a CSR containing the email address before we merge this.

File: bin/request-vsp-certs [hunk 1]
```diff
@@ -8,6 +8,7 @@ fi
 
 ENVIRONMENT_NAME=$1
 VERSION_NUMBER=$2
+EMAIL_ADDRESS="capt-dev@digital.eductation.gov.uk"
```
Comment (mec) [inline bin/request-vsp-certs]: oh no... amazing how far one typo will take you.

File: bin/request-vsp-certs [hunk 2]
```diff
@@ -155,24 +156,29 @@ function request_cert {
...
+  echo "Now enrol the generated $TYPE CSR to Verify."
 }
 
 if ! az account show > /dev/null; then
   echo "Logging in to Azure..."
   az login
 fi
 
+RAILS_ROOT="$(bundle exec rails runner "puts Rails.root")"
+CERTS_DIR="${RAILS_ROOT}/tmp/certs/"
```
Comment (erbridge) [inline bin/request-vsp-certs]: Can we put this setup stuff alongside where we do the other global environment setup at the top of the file?

**Comment by pezholio** [comment]
@mec Can we merge this?

**Comment by erbridge** [comment]
Let's merge this?

**Pull request merged**